src/linux/btop_collect.cpp

```diff
@@ -1865,6 +1866,7 @@ namespace Mem {
 				auto use_fstab = Config::getB("use_fstab");
 				auto only_physical = Config::getB("only_physical");
 				auto zfs_hide_datasets = Config::getB("zfs_hide_datasets");
+				auto only_first_btrfs_subvol = Config::getB("only_first_btrfs_subvol");
```

Comment:
```suggestion
				auto btrfs_group_subvolumes = Config::getB("btrfs_group_subvolumes");
```

maybe? This is more inline with the previous option for `zfs`.